src/frameworks/drupal7/drush.md

```diff
@@ -10,11 +10,41 @@ commands at [Drush.org](http://www.drush.org). If you have never used
 Drush before, you can learn more about it on the [Drush GitHub
 Repository](https://github.com/drush-ops/drush#description)
 
-Platform.sh has Drush installed by default, so all environments can utilize Drush commands in the development process. You can use the CLI to set up Drush aliases, to easily run Drush commands on
-specific remote Platform.sh environments.
+- Platform.sh's Drupal templates have Drush installed automatically.
 
-> **note**
-> Platform's CLI requires **Drush 6 or greater**.
+- All custom environments can utilize Drush commands in the development process if they have `drush` configured as a composer requirement and added it to the environment's `$PATH`. See the section below on [install Drush in custom projects](#install-drush-in-custom-projects)
+
+- In addition, you can use the [Platform.sh CLI](/overview/cli.md) to set up Drush aliases easily for all of your project's environments. See the section below on [use drush aliases](#use-drush-aliases)
+
+## Installing Drush
+
+## Install Drush on Platform.sh-provided Drupal examples
+
+Platform.sh's Drupal templates have Drush installed automatically via Composer and should not require you to add it.
+
+## Install Drush in custom projects
+
+### Make Drush a project requirement
```

Comment:
Could this also describe using the `dependencies` section of the . platform.app.yaml?

Reply:
Well, do we have a preference for dependencies over composer.json?

Reply:
On D8 we favor composer.  On D7, we're using dependencies.